InscryptionAPI/Card/AbilityManager.cs

```diff
@@ -74,6 +74,34 @@ public static void SyncAbilityList()
         AllAbilityInfos = AllAbilities.Select(x => x.Info).ToList();
     }
 
+    public static void RegisterOpposingSlotCountModifier(Ability ab, Func<PlayableCard, int> modifier)
```

Comment:
Same comment as with the Waterborne PR. Since ExtendedAbilityBehaviour is where the API lets you set which slots you attack, we should follow that pattern for this as well to be consistent.

Something like adding this to ExtendedAbilityBehaviour

```c#
public virtual int NumberOfAdditionalSniperAttacks() => 0;
```

instead of requiring them to register a separate function in the AbilityManager.

It doesn't have to be named exactly that or be implemented in exactly that way, but since that's where the API lets you choose which slots you attack, that should also be where the API lets you choose how many sniper attacks you make. It'll keep things consistent.

Reply:
i'll add that support on top of the current method so it also works for non-extendedabilitybehaviour users